addon/components/frost-password.js

```diff
@@ -67,17 +67,12 @@ export default Component.extend(FrostEventsProxy, {
       isRevealed: false,
       revealable: false,
       tabindex: 0,
-      value: null,
```

Comment:
As stated in the referenced issue, I don't see the benefit of initializing to a value of `null`